src/Documentation/Advanced-Concepts/Codegen.md

```diff
@@ -1,16 +1,16 @@
+1.	**Build-time Codegen** - in this mode, codegen will run every time your project is compiled. A build task is injected into your project's build pipeline and the code is generated in the project's intermediate output directory. To activate this mode, add the package `Microsoft.Orleans.OrleansCodeGenerator.Build` to your **Grain Interface** project. If you edit your `.csproj` file, you will see that an extra build target was added. This mode allows a user to step-into the generated code while debugging since the file is physically on the disk. However, your build time will be slower than usual if you have big projects. This is the default mode selected when you create a **Grain Interface** project by using the Visual Studio templates.
 
-2.	**Runtime Codegen** - introduced recently by @reubenbond, different from the build-time, this mode makes `OrleansCodeGenerator` to generate code when the Silo is starting. The good thing is that there is no `.cs` file on disk for the generated code, and you don't have an extra build step on your build pipeline. The bad thing, is that you can't step-into the generated code while debugging and it may increase your silo start time a bit. To enable this mode, if you were using the **build-time** codegen, remove all the references to `Microsoft.Orleans.OrleansCodeGenerator.Build` package and all the `Properties\orleans.codegen.cs` files then add a reference to `Microsoft.Orleans.OrleansCodeGenerator` in your **Silo host** project. If you created the project by using the Visual Studio templates, it should already be there otherwise, you can just install `icrosoft.Orleans.OrleansCodeGenerator` on your silo project. All set! The codegen will run at Silo startup.
+2.	**Runtime Codegen** - This mode makes `OrleansCodeGenerator` to generate code when the Silo is starting. As such, no code is generated during the build. Users can therefore not step-into the generated code while debugging and it will increase silo initialization time. To enable this mode, if you were using the **build-time** codegen, remove the the `Microsoft.Orleans.OrleansCodeGenerator.Build` package and install `Microsoft.Orleans.OrleansCodeGenerator` in your **Silo host** project. If the project was created via the Visual Studio templates then it will already be installed. Otherwise, just install `Microsoft.Orleans.OrleansCodeGenerator` on your silo project.
```

Comment:
This package needs to be added to both the Silo host project, and the client project.